content/learn/contribute/reference/triage.md

```diff
@@ -111,6 +111,8 @@ There are several paths for PRs to be closed:
 4. Anyone may nominate a PR for closure, by bringing it to the attention of the author and / or one of the SMEs / Maintainers. Let them press the button, but this is generally well-received and helpful.
 5. SMEs or Maintainers may, and are encouraged, to unilaterally close PRs that fall into one or more of the remaining categories.
 6. In the case of PRs where some members of the community, other than the author, are in favor, and some are opposed, any two relevant SMEs or Maintainers may act in concert to close the PR.
+7. For a PR that has been sitting for a while and became bitrotten, check with the original author if they intend to continue working on it. If not, or without a response, the PR can be labeled with `S-Adopt-Me`, and closed. Tracking adoption progress will happen in a linked issue.
+8. Inactive `X-Controversial` can be closed if relevant SMEs or Maintainers have decided there's no more interest for it. If it's still interesting and controversial, a decision must be made.
```

Comment:
```suggestion
8. Inactive `X-Controversial` can be closed if relevant SMEs or Maintainers have decided there's no more interest for it. If it's still interesting and controversial, a decision must be made. Depending on the age of the PR, it may be better to move discussion to an issue if there's little engagement with the PR.
```
Less sure about this one, similar motivation though... if we're having trouble getting people to weigh in on a controversial PR, they can hang around indefinitely.

Reply:
I would prefer that maintainers and SMEs reach a decision rather than continue discutions. Worst case is escalating to project lead. It shouldn't happen that often